pallets/Assets/src/lib.rs

```diff
@@ -0,0 +1,89 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use sp_std::prelude::*;
+use frame_system::{self as system, ensure_signed};
+use frame_support::{
+    decl_error, decl_event, decl_module, decl_storage,
+    dispatch::DispatchResult,
+};
+
+use sp_runtime::traits::StaticLookup;
+use sp_core::U256;
+use polkadex_primitives::assets::AssetId;
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod test;
+mod banchmarking;
+pub mod weights;
+pub use weights::WeightInfo;
+
+pub trait Config: system::Config {
+    type Event: From<Event<Self>> + Into<<Self as system::Config>::Event>;
+    type WeightInfo: WeightInfo;
```

Comment:
Use the Balance type as generic and not U256. It will be hard later to do the conventions every single time